### ISSUE TITLE
[Q] Good Work, but how translations work?

### ISSUE THREAD

**Issue by gabrieldejesus** [ptkdev-boilerplate/svelte-electron-boilerplate#3]
### Question

I loved the template excellent work.
I only have one doubt how do the translations work?

**Comment by ptkdev** [comment]
Hi!
Next week I try to update this boilerplate with new examples and code. 
You can see [webcomponents boilerplate](https://github.com/ptkdev-boilerplate/svelte-webcomponent-boilerplate/blob/main/app/html/webcomponent.svelte) (where this feature is updated).

CODE:
```javascript
translate("welcome", { name: "alex" })
```

en.json
```json
{
"welcome": "my name is {{name}}"
}
```

And print "my name is Alex" in HTML. 

In v2.0.0 I import this feature from webcomponent boilerplate. Stay tuned!

**Comment by gabrieldejesus** [comment]
Thanks for the answer @ptkdev now I understand how it works, if you need help with these updates please contact me I will be happy to help with the project

**Comment by ptkdev** [comment]
Available in `v2.0.0` release.
